extensions/linz/README.md

```diff
@@ -22,12 +22,12 @@ extension which adds constraints to default STAC schema properties.
 - [JSON Schema](./schema.json)
 - [Changelog](./CHANGELOG.md)
 
-## Item Fields
+## Item Properties
 
-| Field Name           | Type                | Description                                                                                                                                                  |
-| -------------------- | ------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------ |
-| linz:geospatial_type | string              | **REQUIRED**. A general description of the type of content that can be found in the dataset. See the [list of accepted geospatial types](#geospatial-types). |
-| processing:software  | Map<string, string> | Recommended. The software and versions which were used to generate the dataset. See [reference](https://github.com/stac-extensions/processing).              |
+| Property Name                   | Type                | Description                                                                                                                                     |
```

Comment:
This still seems to be called 'Field Name' in most extension repos I can see.

Reply:
**https://github.com/stac-extensions/table#item-properties-and-collection-fields**